wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -174,5 +175,8 @@
   /** The projects volume has a standard name used in a couple of locations. */
   public static final String PROJECTS_VOLUME_NAME = "projects";
 
+  /** Attribute of {@link Server} that specifies exposure of which port created the server */
+  public static final String SERVER_TARGET_PORT_ATTRIBUTE = "port";
```

Comment:
Should the constant name be updated as well?

Reply:
I'll rename it